transformer/compose/utils.go

```diff
@@ -20,20 +20,34 @@ import (
 	"fmt"
 	"hash/fnv"
 	"os"
+	"path/filepath"
 	"strings"
 
 	"github.com/docker/cli/opts"
 	"github.com/konveyor/move2kube/common"
+	"github.com/konveyor/move2kube/qaengine"
+	irtypes "github.com/konveyor/move2kube/types/ir"
 	"github.com/sirupsen/logrus"
 	core "k8s.io/kubernetes/pkg/apis/core"
 )
 
 const (
 	defaultEnvFile        string = ".env"
 	modeReadOnly          string = "ro"
+	modeReadWrite         string = "rw"
 	tmpFsPath             string = "tmpfs"
 	defaultSecretBasePath string = "/var/secrets"
 	envFile               string = "env_file"
+	maxConfigMapSizeLimit int    = 1024 * 1024
+	// cfgMapPrefix defines the prefix to be used for config maps
+	cfgMapPrefix   = "configmap"
+	secretPrefix   = "secret"
+	volQaPrefixKey = "move2kube.storage.type"
```

Comment:
we generally keep this in the constants file.

Reply:
Done